i18n/index.ts

```diff
@@ -1,14 +1,17 @@
 import en from './locales/en.json'
 import ja from './locales/ja.json'
 import pt from './locales/pt.json'
+import ger from './locales/ger.json'
```

Comment:
Since the locale for German is `de_DE` I think this import should be:

`import de from ./locales/de.json`